pages/cart.tsx

```diff
@@ -96,11 +154,47 @@ const Cart = () => {
                               >
                                 <span>Remove</span>
                               </button>
+                              {quantityErrors && (
+                                <div>
+                                  {quantityErrors.map((error: any) => (
+                                    <span
+                                      className="text-red-500 text-sm font-medium"
+                                      key={error.field}
+                                    >
+                                      {error.message}
+                                    </span>
+                                  ))}
+                                </div>
+                              )}
+                            </div>
+                            <div className="flex justify-items-end space-x-4">
+                              <input
+                                type="number"
+                                className={clsx(
+                                  "h-8 w-16 block border-gray-300 rounded-md shadow-sm sm:text-sm",
+                                  quantityErrors && "border-red-500"
+                                )}
+                                value={
+                                  quantityForLine?.find(
+                                    (q) => q?.lineId === lineID
+                                  )?.quantity
+                                }
+                                onFocus={() => setQuantityErrors(null)}
+                                onChange={(ev) => changeLineState(ev, lineID)}
+                                onBlur={(ev) => onQuantityUpdate(ev, lineID)}
+                                onKeyPress={(ev) => {
+                                  if (ev.key === "Enter") {
```

Comment:
Does it also work on mobile?

I'm not sure about this, but can we wrap input in the `form` tag, then defining "onKeyPress" wouldn't be necessary,  only onSubmit would be needed